RevenueCatUI/Templates/Template2View.swift

```diff
@@ -83,20 +86,35 @@ struct Template2View: TemplateViewType {
                 Spacer()
             }
 
-            if self.configuration.mode.shouldDisplayPackages {
-                self.packages
+            VStack {
+                self.packagesScrollHack
+
                 Spacer()
-            } else {
-                self.packages
-                    .onSizeChange(.vertical) { if $0 > 0 { self.containerHeight = $0 } }
-                    .hideFooterContent(self.configuration,
-                                       hide: !self.displayingAllPlans,
-                                       offset: self.containerHeight)
             }
+            // Sets a max footer height so there is enough room for paywall content above
+            .frame(maxHeight: configuration.mode == .fullScreen ? nil : Constants.maxFooterHeight)
+            .hideFooterContent(self.configuration,
+                               hide: !self.configuration.mode.shouldDisplayPackages && !self.displayingAllPlans)
         }
         .frame(maxHeight: .infinity)
     }
 
+    /// Hack because scrollableIfNecessary() doesn't work on iOS 15 because of unknown reason
+    private var packagesScrollHack: some View {
+        Group {
```

Comment:
The recommended way of doing this is:
```suggestion
    @ViewBuilder
    private var packagesScrollHack: some View {
```